### PR TITLE
edits to authorcontent.Rmd

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

File: authorcontent.Rmd [hunk 1]
```diff
@@ -21,8 +21,8 @@ Both are given the same [promotional treatment](#promote) by rOpenSci.
 
 ## Themes of posts {#postthemes}
 
-Read a few [blog posts and tech notes](https://ropensci.org/archive/) and consider what you like (or not) about them.
-Most posts written by community members are about packages that have passed rOpenSci software peer review. However, we have posts on a range of topics and we encourage you to consider these or to propose others.
+Read a few [posts](/archive/) and consider what you like (or not) about them.
```
Comment (maelle) [inline authorcontent.Rmd]: The relative link won't work from the blog guide subdomain.

Comment (stefaniebutland) [reply authorcontent.Rmd]: fixed

Comment (maelle) [reply authorcontent.Rmd]: it doesn't look fixed?

Comment (maelle) [inline authorcontent.Rmd]:
```suggestion
Read a few [posts](https://ropensci.org/archive/) and consider what you like (or not) about them.
```

File: authorcontent.Rmd [hunk 2]
```diff
@@ -21,8 +21,8 @@ Both are given the same [promotional treatment](#promote) by rOpenSci.
 
 ## Themes of posts {#postthemes}
 
-Read a few [blog posts and tech notes](https://ropensci.org/archive/) and consider what you like (or not) about them.
-Most posts written by community members are about packages that have passed rOpenSci software peer review. However, we have posts on a range of topics and we encourage you to consider these or to propose others.
+Read a few [posts](https://ropensci.org/archive/) and consider what you like (or not) about them.
```
Comment (maelle) [inline authorcontent.Rmd]: I'm so glad we have the single archive :wink:

**Pull request merged**